pilot/pkg/bootstrap/server.go

```diff
@@ -107,9 +107,9 @@ var (
 	DefaultPlugins = []string{
 		plugin.Authn,
 		plugin.Authz,
+		plugin.Mixer,
 		plugin.Envoyfilter,
 		plugin.Health,
```

Comment:
I *feel* like having this happen after `Health` was intentional. This prevents policy/monitoring on health checks (and prometheus scrapes), I believe.

Do I have that right, @objectiser  ?

Reply:
Move health to top. Its to prevent tracing for health check calls.

Reply:
Can you summarize what health does? Does it just put some header? Does it block requests?

Reply:
Yes the health check filter disables tracing - so needs to occur before the mixer.

@kyessenov  The health check filter just disables sampling of the active span, which is then propagated to any child spans within the proxy or called services.

Reply:
Ok, moved health up prior to mixer.